### PR TITLE
vote: liuzix as tikv reviewer

### PR DESCRIPTION
**Pull request by overvenus**

### CODE REVIEW

**Review by BusyJay — APPROVED** [review]

**Comment by BusyJay** [comment]
Thanks for your contributions! @liuzix

**Review by hicqu — APPROVED** [review]

File: votes/0137-liuzix-as-tikv-committer.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# A Vote for kennytm as TiKV Reviewer
```
Comment (kennytm) [inline votes/0137-liuzix-as-tikv-committer.md:1]: 🤔

Comment (tisonkun) [reply votes/0137-liuzix-as-tikv-committer.md:1]:
```suggestion
# A Vote for liuzix as TiKV Reviewer
```

File: votes/0137-liuzix-as-tikv-committer.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[@liuzix](https://github.com/liuzix) has been working on `tikv/tikv` for more than one years. He has continuously contributed to cdc, and other data migration tools for TiKV. His contributions are listed:
+
+* [Authored pull requests](https://github.com/tikv/tikv/commits?author=liuzix).
+* [Reviewed pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+reviewed-by%3Aliuzix)
+
+I (@overvenus) hereby nominate @liuzix as TiKV Reviewer and call for a vote.
+
+## Result
+
+Not yet.
```
Comment (tisonkun) [inline votes/0137-liuzix-as-tikv-committer.md:14]:
```suggestion
Approved by 2 binding votes, 1 non-binding votes.

* BusyJay (binding)
* hicqu (binding)
* overvenus (non-binding)
```

**Review by tisonkun — COMMENTED** [review]
3 days elapsed and we have already 2 binding votes.

@overvenus I'd suggest you update the content and @BusyJay or @hicqu can help trigger a merge.

**Comment by tisonkun** [comment]
@hicqu would you please help on merging?

**Pull request merged**